CMakeLists.txt

```diff
@@ -66,6 +66,9 @@ if (MSVC)
 
     # Use statically linked runtime
     set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreaded$<$<CONFIG:Debug>:Debug>)
+
+    # Set startup project in Visual Studio
+    set_property( DIRECTORY PROPERTY VS_STARTUP_PROJECT all_tests)
```

Comment:
Could you please tell why we should start from all_tests and not e.g. sc2lib?

Also, this string contains extra space after the opening `(`.

Reply:
The logic was after downloading and building the project, the next step was to run all_tests.

It was an attempt to 'simplify' the process of getting something to show up as a newbie using Visual Studio.  
All the 'VS_****_****' commands were to facilitate using the IDE buttons, once entering Visual Studio, and not using the shell. 
However, one using Visual Studio will eventually have to learn to manually modify these values in the IDE, so perhaps these initial quality-of-life modifications aren't needed.  I was on a bit of an 'automation' kick that day, learning more about CMAKE.